### DISCUSSION TITLE
TransCopy (.TC) floppy image support?

### DISCUSSION THREAD

**Discussion by n1tr0ms** [joncampbell123/dosbox-x#5091] [Ideas]
This format contains floppy data as is which makes possible to play many early games with copy protection.
The only known build of dosbox with support of this format is Dosbox SVN fork by NewRisingSun known as DOSBox-TC. Version from 06.02.2019 is avaliable at eXoDOS collection.
With dosbox-x save states option it could make one of the best ways to play different early titles.

**Comment by Torinde** [comment]
Related:
- [tc_dosbox.zip](https://github.com/dosbox-staging/dosbox-staging/files/8693734/tc_dosbox.zip)
- https://github.com/exoscoriae/eXoDOS/issues/1586
- https://github.com/dosbox-staging/dosbox-staging/issues/2224
